packages/edit-widgets/src/components/error-boundary/style.scss

```diff
@@ -2,6 +2,6 @@
 	margin: auto;
 	max-width: 780px;
 	padding: 20px;
-	margin-top: 60px;
+	margin-top: $header-height;
```

Comment:
This is the top margin of the content that will be displayed when a fatal error occurs in the block editor. This value was originally hardcoded, but it should be expected to match the height of the header.

![error-boundary](https://github.com/WordPress/gutenberg/assets/54422211/a6edf495-af5a-44fa-97d2-96cdfac621e2)